examples/phenology-uc2/README.md

```diff
@@ -0,0 +1,5 @@
+# Phenology example
+
+This example uses a Jupyter notebook to show a complete workflow based on openEO. The main file is the notebook: Phenology.ipynb, the 'udf' folder contains user defined functions used by the notebook.
```

Comment:
Having clickable links would be great. Also, what are the other files in the folder (e.g. Timeseries batch compute.ipynb) useful for? I'd either describe or remove them?